GenumKit/Parsers/StoryboardParser.swift

```diff
@@ -84,6 +84,7 @@ public final class StoryboardParser {
     storyboardsSegues[storyboardName] = delegate.segues
   }
 
+  @available(OSX 10.10, *)
```

Comment:
Why did you add this restriction? This prevented me from compiling the SwiftGen OS X executable when I was trying to release so I removed it, but wanted to be sure what the primary intent was.
People even not in OS X 10.10 should still be able to parse iOS storyboards for example.